generate_R_table.py

```diff
@@ -68,8 +69,13 @@ def main():
     # answers to all the questions
     # Key = item number;
     # value = (correct_button_code, LeftorRight)
-    answer_key = dict_from_table(read_table(file_names['Question key filename']),
-                                                    paired=False)
+    
+    # THE TWO LINES BELOW ARE COMMENTED BECAUSE THEY DON'T WORK CORRECTLY.
+    # THE LINE BELOW THEM IS A QUICK FIX (ALSO SEE LINE 396). 
+    #answer_key = dict_from_table(read_table(file_names['Question key filename']),
+    #                                               paired=False)
```

Comment:
At the time I was using PDB in  command line. As far as I remember, my problem was that when I wanted to look up certain variables' values at runtime, it would just tell me that it's a generator object. If I ever return to that, I might try some IDE like PyCharm and see whether it can show generator's contents